calendar/Module.md

```diff
@@ -0,0 +1,94 @@
+## Overview
+Ballerina connector for Microsoft Outlook Calendar is connecting the Calendar API in Microsoft Graph via Ballerina language easily. 
```

Comment:
Shall we remove unnecessary spaces and rephrase this as: "Ballerina connector for Microsoft Outlook Calendar is connecting the Calendar API in Microsoft Graph via Ballerina language easily. It provides the capability to perform CRUD (Create, Read, Update, and Delete) operations on  [Event](https://docs.microsoft.com/en-us/graph/api/resources/event?view=graph-rest-1.0) & [Calendar](https://docs.microsoft.com/en-us/graph/api/resources/event?view=graph-rest-1.0)  stored in Microsoft OneDrive. 	 
  
This module supports the Microsoft Outlook Calendar API v1.0 version."

Reply:
Rephrased as above /Thanks